ocs_ci/ocs/resources/objectbucket.py

```diff
@@ -383,9 +383,16 @@ def internal_delete(self):
             NotFoundError: In case the OBC was not found
 
         """
-        result = self.mcg.exec_mcg_cmd(f"obc delete {self.name}")
-        if "deleting" and self.name not in result.stderr.lower():
-            raise NotFoundError(result)
+        try:
+            result = self.mcg.exec_mcg_cmd(f"obc delete {self.name}")
+            if "deleting" and self.name not in result.stderr.lower():
```

Comment:
Daniel found some odd things in this condition, which was here even you touched the code.

The string "deleting" will be evaluated as True always.

So it should be probably defined like:
```
if "deleting" not in result.stderr.lower() and self.name not in result.stderr.lower():
    raise NotFoundError(result)
```